crates/astria-sequencer/src/app.rs

```diff
@@ -448,6 +461,7 @@ impl App {
 
         // Get the latest version of the state, now that we've committed it.
         self.state = Arc::new(StateDelta::new(storage.latest_snapshot()));
+        self.is_proposer = false;
```

Comment:
Add a comment for why this is necessary to if the reader forgot what `is_propser` is.